network-api/networkapi/wagtailpages/tests/buyersguide/base.py

```diff
@@ -4,7 +4,10 @@
 from django.test.utils import override_settings
 
 from networkapi.wagtailpages.factory.homepage import WagtailHomepageFactory
+from networkapi.wagtailpages.factory.buyersguide import BuyersGuideCallToActionFactory
+
 from networkapi.wagtailpages.pagemodels.base import Homepage
+
```

Comment:
**Question** What is purpose of these blank lines?

Reply:
Per our discussion, removed!